### ISSUE TITLE
Improved Consistency using DDIM inversion (?)

### ISSUE THREAD

**Issue by HyeonHo99** [showlab/Tune-A-Video#26]
Hi, thank you so much for impressive works!

I noticed there was an updated 'News' of 'Improved Consistency through DDIM inversion'.
Can you explain a bit more about this update? So what I understood is "before : DDPM inversion (DDPM forward and reverse)" then "after: DDIM inversion (DDIM forward and reverse).
Am I right?
Also, then is DDIM sampler used in both fine-tuning an inference?

Thank you again for nice works.

**Comment by HyeonHo99** [comment]
Plus)
In your train_tuneavideo.py, it still uses DDPMScheduler
ex>>noise_scheduler = DDPMScheduler.from_pretrained(pretrained_model_path, subfolder="scheduler")
Is this python file just not updated yet?

Thank you

**Comment by Randle-Github** [comment]
I've checked that it is the problem of attention_mask, which is not actually used in training process. Just delete it and it could be working. However, there're other problems about the version, maybe it is better to alter to the right verision.

**Comment by zhangjiewu** [comment]
the DDIM inversion is only used at inference

before: DDIM sampling from random noise
after: DDIM sampling from inverted noise of source video
